rewrite-weblogic/docs/procedures/index.md

```diff
@@ -4,6 +4,9 @@ Depending on your application dependencies and WebLogic Server and JDK source ve
 
 You can run Rewrite WebLogic recipes on your code folder using the Maven or Gradle CLI, or include them as a build plug-in in your `pom.xml` file. After running recipes, you are notified of all of the files that have been changed. To see what's changed in the code, run `git diff` or use your preferred IDE's diff viewer. From there, you can commit the changes or add additional recipes based on your needs.
 
+> [!TIP]  
+> When using Maven to run the recipe, you need the current version of the WebLogic Maven plug-in (for example, 14.1.1). After migrating (for example, to 15.1.1), then you need to install the 15.1.1 plug-in version (and sync) to be able to build, compile, or package the migrated application.

```

Comment:
Not all users will need that, only if they have weblogic-specific libraries/plugins on the pom.xml file that this is needed.